### PR TITLE
Closes #5: add github workflow for running unittests

### LINKED ISSUE #5: add CI for python unittests using GitHub Actions
Should be doable with https://github.com/actions/checkout and https://github.com/actions/setup-python

### PR DESCRIPTION
**Pull request by capsulecorplab**
PR closes #5.

### CODE REVIEW

**Comment by capsulecorplab** [comment]
Somethings wrong; The build failed at first, but after re-running with no code changes, it passed.

**Comment by capsulecorplab** [comment]
Refactored the github workflow (i.e., `unittest.yml`) to use the **ubuntu:latest** docker image to install python, activate virtual environment, install python dependencies, & run unit tests. This should be a more reliable CI pipeline that doesn't cache anything between builds.

**Comment by fpdotmonkey** [comment]
What should I do to verify that this is working?

File: .github/workflows/unittests.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+jobs:
+  deploy:
+    runs-on: ubuntu-18.04
+    steps:
+      - name: Checkout GitHub workspace
+        uses: actions/checkout@v2
+
+      - name: Install python, activate venv, install dependencies, & run tests all in docker container
+        run: sudo docker run --rm -v $PWD:/src -w "/src" ubuntu:latest bash -c "apt update && apt -y upgrade && apt install -y python3-pip python3-venv && ./setup.sh && source ./venv/bin/activate && ./runtests.sh"
```
Comment (capsulecorplab) [inline .github/workflows/unittests.yml:14]: To verify if it's working, you can run `docker run --rm -v $PWD:/src -w "/src" ubuntu:latest bash -c "apt update && apt -y upgrade && apt install -y python3-pip python3-venv && ./setup.sh && source ./venv/bin/activate && ./runtests.sh"` on your local machine in the project-tetra-display root directory to emulate what's happening in the CI build, or look at the recent build itself https://github.com/tetrabiodistributed/project-tetra-display/pull/7/checks?check_run_id=941830230. This PR is more about validating whether this is the automated workflow we actually want in a CI build.

**Pull request merged**